### ISSUE TITLE
Drawer - Heading element not being read aloud

### ISSUE THREAD

**Issue by joe-replin** [adaptlearning/adapt-contrib-core#369]
### Subject of the issue/enhancement/features
 At the end of the side drawer, the heading element is receiving focus but is not being read aloud in JAWS & Firefox. 

[Code Reference]
`<span id="drawer-heading" class="aria-label">Side drawer</span>`

### Your environment
* Adapt v5.30.3
* Firefox v112.0.2
* Windows 11
* JAWS 2022.2207.25

**Comment by oliverfoster** [comment]
Which heading element? Can you provide a screenshot of the dom or ui with the heading element please?

**Comment by joe-replin** [comment]
Sorry, needed to change my original ticket to the code snippet in order to see it. Here are some screenshots.
![image](https://user-images.githubusercontent.com/7740674/236062588-5a3bf489-dfb3-46f0-a797-509a02a4d25b.png)
![image](https://user-images.githubusercontent.com/7740674/236062626-9aebb1e0-bbf9-4397-904b-93164fb817ed.png)

**Comment by oliverfoster** [comment]
When is it not being read aloud? When arrowing through the drawer with the screen reader cursor or when opening the drawer?

It will only receive focus proper when the drawer is opened and then should be read. It should also read when the screen reader cursor lands on it otherwise.

Slight discrepancy: "At the end of..." Is not where that element lives. Could they perhaps be talking about the focus wrap element?

**Comment by joe-replin** [comment]
In Chrome, it is being read as intended under the following circumstances.

In Firefox, this is the behavior I'm seeing:
- Upon opening of the drawer, my browser tabs are instead announced. Selecting `tab` from there shifts focus to the drawer close button.
- After I tab through each list item in the drawer, focus is given to the heading (as seen in my screenshot above) but nothing is being read.

**Comment by oliverfoster** [comment]
Works in firefox 113.0.1, which is the current beta. 
It's a bug in firefox that doesn't read the dialog heading properly.

This behaviour is implemented correctly as per the spec 
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/dialog_role

**Comment by joe-replin** [comment]
Testing results.
Chrome & NVDA - Drawer heading is read out. Behavior is as expected.
Chrome & JAWS - Drawer heading is read out. Behavior is as expected.
Firefox & NVDA - Drawer heading is read out. Behavior is as expected.
Firefox & JAWS - See previous comment. https://github.com/adaptlearning/adapt-contrib-core/issues/369#issuecomment-1546165974

**Comment by joe-replin** [comment]
Updated to Firefox v113 and this issue has been resolved.
